Cargo.toml

```diff
@@ -27,3 +27,4 @@ alloc = []
 random = ["rand"]
 serde = ["dep:serde", "alloc"]
 std = ["alloc", "serde?/std"]
+unsafe_rust = []
```

Comment:
Maybe just calll `unsafe` for the sake of simplicity?

Reply:
This is now `optimized` to match the issue.